src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -426,7 +426,11 @@
                  :error-message err-text :disabled-reason :error)
           (swap! state update :server-response assoc
                  :billing-projects (map :projectName projects)
-                 :disabled-reason (if (empty? projects) :no-billing nil))))))})
+                 :disabled-reason (if (empty? projects) :no-billing nil)))))
+     (utils/ajax
+      {:url (config/featured-json-url)
+       :on-done (fn [{:keys [get-parsed-response]}]
+                  (swap! state update :server-response assoc :featured-workspaces (set (get-parsed-response))))}))})
```

Comment:
I think this ajax call should be pretty resilient in the case that the user editing/uploading the featured.json made a mistake. Could you trap for non-success responses and for malformed json, and in those cases use the empty set? If this is a bunch of work, I'd be okay calling it scope creep and not doing it.